src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final int RESIZE = 2;
+    private static final double THRESHOLD = 0.75;
```

Comment:
wrong naming, 0.75 it's a load factor, the threshold is a (table.length * loadfactor)